docs/fiddles/communication/two-processes/synchronous-messages/main.js

```diff
@@ -0,0 +1,29 @@
+const { app, BrowserWindow, ipcMain } = require('electron')
+
+let mainWindow = null
+
+function createWindow () {
+  const windowOptions = {
+    width: 600,
+    height: 400,
+    title: 'Synchronous',
```

Comment:
```suggestion
    title: 'Synchronous messages',
```

Reply:
Okay. I will do this. Also for some reason the tests are failing on Circle CI Linux. What am I doing wrong?

Reply:
@97amarnathk don't worry about the failing tests on CircleCI Linux.  They are unrelated to your change.

Reply:
I have added the suggested changes.